Could not install

### Describe the bug

`src/bitutils.c:1:10: fatal error: postgres.h: No such file or directory` during build step.

### Context

YunoHost 12.0.9.2 in LXC

### Logs

https://paste.yunohost.org/raw/muvipayozu

> src/bitutils.c:1:10: fatal error: postgres.h: No such file or directory

postgres.h is provided by postgresql-server-dev-*, and postgresql-server-dev-15 is installed during your test.

Let see what the CI result will be

--- a/README.md
+++ b/README.md
@@ -21,7 +21,7 @@ It shall NOT be edited by hand.
 AppFlowy is the AI workspace where you achieve more without losing control of your data
 
 
-**Shipped version:** 0.9.18~ynh1
+**Shipped version:** 0.9.18~ynh2
 
 ## Screenshots
 

--- a/README_es.md
+++ b/README_es.md
@@ -21,7 +21,7 @@ No se debe editar a mano.
 AppFlowy is the AI workspace where you achieve more without losing control of your data
 
 
-**Versión actual:** 0.9.18~ynh1
+**Versión actual:** 0.9.18~ynh2
 
 ## Capturas
 

--- a/README_eu.md
+++ b/README_eu.md
@@ -21,7 +21,7 @@ EZ editatu eskuz.
 AppFlowy is the AI workspace where you achieve more without losing control of your data
 
 
-**Paketatutako bertsioa:** 0.9.18~ynh1
+**Paketatutako bertsioa:** 0.9.18~ynh2
 
 ## Pantaila-argazkiak
 

--- a/README_fr.md
+++ b/README_fr.md
@@ -21,7 +21,7 @@ Il NE doit PAS être modifié à la main.
 AppFlowy is the AI workspace where you achieve more without losing control of your data
 
 
-**Version incluse :** 0.9.18~ynh1
+**Version incluse :** 0.9.18~ynh2
 
 ## Captures d’écran
 

--- a/README_gl.md
+++ b/README_gl.md
@@ -21,7 +21,7 @@ NON debe editarse manualmente.
 AppFlowy is the AI workspace where you achieve more without losing control of your data
 
 
-**Versión proporcionada:** 0.9.18~ynh1
+**Versión proporcionada:** 0.9.18~ynh2
 
 ## Capturas de pantalla
 

--- a/README_id.md
+++ b/README_id.md
@@ -21,7 +21,7 @@ Ini TIDAK boleh diedit dengan tangan.
 AppFlowy is the AI workspace where you achieve more without losing control of your data
 
 
-**Versi terkirim:** 0.9.18~ynh1
+**Versi terkirim:** 0.9.18~ynh2
 
 ## Tangkapan Layar
 

--- a/README_nl.md
+++ b/README_nl.md
@@ -21,7 +21,7 @@ Hij mag NIET handmatig aangepast worden.
 AppFlowy is the AI workspace where you achieve more without losing control of your data
 
 
-**Geleverde versie:** 0.9.18~ynh1
+**Geleverde versie:** 0.9.18~ynh2
 
 ## Schermafdrukken
 

--- a/README_pl.md
+++ b/README_pl.md
@@ -21,7 +21,7 @@ Nie powinno być ono edytowane ręcznie.
 AppFlowy is the AI workspace where you achieve more without losing control of your data
 
 
-**Dostarczona wersja:** 0.9.18~ynh1
+**Dostarczona wersja:** 0.9.18~ynh2
 
 ## Zrzuty ekranu
 

--- a/README_ru.md
+++ b/README_ru.md
@@ -21,7 +21,7 @@
 AppFlowy is the AI workspace where you achieve more without losing control of your data
 
 
-**Поставляемая версия:** 0.9.18~ynh1
+**Поставляемая версия:** 0.9.18~ynh2
 
 ## Снимки экрана
 

--- a/README_zh_Hans.md
+++ b/README_zh_Hans.md
@@ -21,7 +21,7 @@
 AppFlowy is the AI workspace where you achieve more without losing control of your data
 
 
-**分发版本：** 0.9.18~ynh1
+**分发版本：** 0.9.18~ynh2
 
 ## 截图
 

--- a/manifest.toml
+++ b/manifest.toml
@@ -6,7 +6,7 @@ id = "appflowy"
 name = "AppFlowy"
 description.en = "Bring projects, wikis, and teams together with AI"
 
-version = "0.9.18~ynh1"
+version = "0.9.18~ynh2"
 
 maintainers = ["yalh76"]
 

--- a/scripts/install
+++ b/scripts/install
@@ -146,8 +146,10 @@ ynh_script_progression "Building app..."
 
 # Installing pgvector
 pushd "$install_dir/build/pgvector"
-	make
-	make install
+    export PG_MAJOR_VERSION=$(sudo --login --user=postgres psql -t -c "SELECT current_setting('server_version_num')::int / 10000;" | xargs)
+    export PG_CONFIG="/usr/lib/postgresql/$PG_MAJOR_VERSION/bin/pg_config"
+    make
+    make install
 popd
 
 ynh_safe_rm "$install_dir/build/"

